testomania/src/main/java/com/earth/testomania/common/unsplash/UnsplashApi.kt

```diff
@@ -0,0 +1,22 @@
+package com.earth.testomania.common.unsplash
+
+import com.earth.testomania.BuildConfig
+import com.earth.testomania.common.unsplash.models.UnsplashDto
+import retrofit2.Response
+import retrofit2.http.GET
+import retrofit2.http.Headers
+import retrofit2.http.Query
+
+interface UnsplashApi {
```

Comment:
Please pull my changes and let's put Apis together.
I created an `image` folder under apis folder, pleas put this API there, similarly to Quiz API
![image](https://user-images.githubusercontent.com/6556678/208994109-4becb7e1-3798-427f-9f1a-9712072b30b8.png)